test/Motor.Extensions.Hosting_UnitTest/MultiOutputServiceAdapterTests.cs

```diff
@@ -29,7 +29,7 @@ public async Task HandleMessageAsync_WithContextAndInput_HasContext()
         var context = CreateMotorEvent("message");
         var messageHandler = GetMessageHandler(service: converterMock.Object);
 
-        await messageHandler.HandleMessageAsync(context).ConfigureAwait(false);
+        await messageHandler.HandleMessageAsync(context);
```

Comment:
`ConfigureAwait(false)` has been removed in tests due to [xUnit1030](https://xunit.net/xunit.analyzers/rules/xUnit1030).